pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -591,6 +741,11 @@ func (lbaas *LbaasV2) UpdateLoadBalancer(clusterName string, service *api.Servic
 			if err != nil {
 				return err
 			}
+			if lbaas.opts.ManageSecurityGroups {
```

Comment:
Why is this here?  Won't the security group already have been created in `EnsureLoadBalancer`?